tests/test_public_hist.py

```diff
@@ -690,14 +695,20 @@ def ar(*args):
     with pytest.raises(KeyError):
         a.fill((1, 2), foo=(1, 1))
     with pytest.raises(ValueError):
-        a.fill((1, 2), weight=(1,))
```

Comment:
I am not sure why this changed. I think it should still fail.

Reply:
`a.fill((1, 2), weight=1)` is allowed, but `a.fill((1, 2), weight=(1,))` should not be allowed

Reply:
The good thing is:

```python
a.fill((1,2,3), weight=(1,2))
```

still throws, so this is just "broadcasting", but not sure why it started doing so.